### PR TITLE
Offmed Port - Fishmed

### PR DESCRIPTION
**Pull request by imatsoup**
## General information

dont get excited yet yall, this is mostly here so I can keep track of the diff easier and have aku put it on the dev server. 
## Why / Balance

shiny new medical system
## Technical details

- [x] Resprited X and Y to work with this change.
- [x] Implemented Z and X to handle Y edge case.
- [ ] Added a new component for all X to prevent Z.

## Test plan

suffering
## Breaking changes

everything, just all of it
## Changelog
:cl: 

- add: Added braindeath
- remove: lamestream medical

### CODE REVIEW

**Comment by imatsoup** [comment]
ftr i will obviously be rewriting the pull request when it comes time to actually do this

File: Content.Client/Body/VisualBodySystem.cs [hunk 1]
```diff
@@ -32,6 +34,13 @@ public override void Initialize()
 
         SubscribeLocalEvent<VisualOrganMarkingsComponent, BodyRelayedEvent<HumanoidLayerVisibilityChangedEvent>>(OnMarkingsChangedVisibility);
 
+        // Begin Offbrand
+        SubscribeLocalEvent<VisualOrganComponent, BodyRelayedEvent<BodyAppearanceRelayTargetAddedEvent>>(OnVisualRelayTargetAdded);
+        SubscribeLocalEvent<VisualOrganComponent, BodyRelayedEvent<BodyAppearanceRelayTargetRemovedEvent>>(OnVisualRelayTargetRemoved);
+        SubscribeLocalEvent<VisualOrganMarkingsComponent, BodyRelayedEvent<BodyAppearanceRelayTargetAddedEvent>>(OnMarkingsRelayTargetAdded);
+        SubscribeLocalEvent<VisualOrganMarkingsComponent, BodyRelayedEvent<BodyAppearanceRelayTargetRemovedEvent>>(OnMarkingsRelayTargetRemoved);
```
Comment (SnappingOpossum) [inline Content.Client/Body/VisualBodySystem.cs]: Use attribute

**Comment by Atakku** [comment]
fihmed

**Comment by DahBrass** [comment]
fihmed

File: Content.Client/_Offbrand/Analyzer/VitalsAnalyzerControl.xaml.cs [hunk 2]
```diff
@@ -63,6 +75,30 @@ public void Update((VitalsData, EntityUid, bool)? scan)
...
+                state = "Minor Radiation Sickness";
+                break;
+            case "StatusEffectModerateRadSickness":
+                RadThresholdState.FontColorOverride = Color.Yellow;
+                state = "Moderate Radiation Sickness";
+                break;
+            case "StatusEffectSevereRadSickness":
+                RadThresholdState.FontColorOverride = Color.Red;
+                state = "Severe Radiation Sickness";
+                break;
```
Comment (SnappingOpossum) [inline Content.Client/_Offbrand/Analyzer/VitalsAnalyzerControl.xaml.cs:97]: Do not put raw strings for these ids.
